docs/glossary.md

```diff
@@ -0,0 +1,60 @@
+---
+id: glossary
+title: GLOSSARY
+sidebar_label: Glossary
+slug: glossary
+---
+
+# 🤸Shenanigan's Glossary🤸
+
+1. **APY:** The annual percentage yield (APY) is the real rate of return earned on a savings deposit or investment taking into account the effect of compounding interest.
+
+2. **Blockchain:** A blockchain is a digital ledger of transactions that are duplicated and distributed across the entire network of computer systems on the blockchain. Each block in the chain contains a number of transactions, and every time a new transaction occurs on the blockchain, a record of that transaction is added to every participant’s ledger. The decentralised database managed by multiple participants is known as Distributed Ledger Technology (DLT).
+
+2. **Challenges:** 
+
+2. **DAO:** A Decentralized Autonomous Organization (DAO) can be described as an open-source blockchain protocol governed by a set of rules, created by its elected members, that automatically execute certain actions without the need for intermediaries.
+
+2. **dApp:** Decentralized applications (dApps) are digital applications or programs that exist and run on a blockchain or P2P network of computers instead of a single computer, and are outside the purview and control of a single authority. 
+
+2. **Dathlete:** Dapp athletes, e-Sport's players, and those who are willing to push themselves to their limits. These Dathletes are backed and fueled by the power of web 3.0 
+
+2. **DeFi:** Decentralized finance (Defi) aims to use technology to remove finacial intermediaries and centralized systems between parties in a financial transaction. 
+
+3. **Dripp.farm:** Shenanigan's first NFT farm.
```

Comment:
I would leave this off this wiki. Dripp.farm can have its own separate wiki. Let keep this pertinent toward Shenanigan.

Reply:
I totally agree with you Victor. I want to see soon, a second version of the dripp.farm